src/Kdyby/Translation/Translator.php

```diff
@@ -130,6 +137,7 @@ public function translate($message, $count = NULL, $parameters = array(), $domai
 			return $message;
 
 		} elseif ($message instanceof Nette\Utils\Html) {
+			$this->logger->log($message, static::LOGGER_NAMESPACE);
```

Comment:
definitely not :) 

but the following code marks message as untraslated, so I presumed logging in this place is also desired..